File: Backend/src/main/java/com/feedle/feedleapi/Networking/SendMessageRequest.java
package com.feedle.feedleapi.Networking;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.feedle.feedleapi.Models.Comment;
import com.feedle.feedleapi.Models.Message;

public class SendMessageRequest extends Request {
    private Message message;
    public SendMessageRequest(Message message)
    {
        super(RequestType.SendMessage);
        this.message = message;
    }

    public Message getMessage() {
        return message;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/SendMessageRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class SendMessageRequest : Request
    {
        [JsonPropertyName("message")]
        public Message Message { get; set; }
        public SendMessageRequest(Message message) : base(RequestType.SendMessage)
        {
            this.Message = message;
        }
    }
}